Netskope/netskope_events/ingest/parser.yml

```
@@ -106,6 +106,9 @@ stages:
         mapping:
           parsed_event.message.alert: event.kind
         fallback: "event"
+      - set:
+          url.original: "https://{{parsed_event.message.url}}"
```

According to the ECS specification, [`url.original`](https://www.elastic.co/guide/en/ecs/8.17/ecs-url.html#field-url-original) is:
> Unmodified original url as seen in the event source

I suggest using `url.full` here instead:
```suggestion
          url.full: "https://{{parsed_event.message.url}}"
```